ts/features/bonus/bpd/components/superCashbackRanking/UserPositionItem.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { profileNameSelector } from "../../../../../store/reducers/profile";
+import { formatNumberWithNoDigits } from "../../../../../utils/stringBuilder";
+import RankPositionItem from "./RankPositionItem";
+
+type Props = {
+  transactionsNumber: number;
+  superCashbackAmount: number;
+  userPosition: number;
+  hideBadge?: boolean;
+} & ReturnType<typeof mapStateToProps>;
+
+const UserPositionItem: React.FunctionComponent<Props> = (props: Props) => (
+  <RankPositionItem
+    transactionsNumber={props.transactionsNumber}
+    superCashbackAmount={props.superCashbackAmount}
+    currentUserPosition={true}
+    boxedLabel={"TU"}
```

Comment:
?

Reply:
I missed the localization of this label ty for the catch!